src/components/AddBook.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
-import { addBook } from '../redux/books/books';
 import createBook from '../redux/books/createBook';
+import { addBooksThunk } from '../redux/books/apiConnect';
 
 export default function Form() {
   const dispatch = useDispatch();
 
-  const newBook = (e) => {
+  const newBook = async (e) => {
     e.preventDefault();
     const { target } = e;
     const title = target.title.value;
     const author = target.author.value;
-    dispatch(addBook(createBook(title, author)));
+    dispatch(addBooksThunk(createBook(title, author)));
     target.title.value = '';
     target.author.value = '';
   };
```

Comment:
- [ ] When I add a book I have to reload the page for the newly added book to reflect on the page from the API. Kindly refactor your code and ensure that when a book is added it is not only added to the API but also on the page immediately without page refresh.